WalletWasabi.Fluent/Controls/CurrencyEntryBox.axaml.cs

```diff
@@ -255,10 +255,10 @@ private bool ValidateEntryText(string preComposedText)
 
```

Comment:
L295: My VS shows an analyzer error on line:

```cs
text = text.Substring(0, 17 + 4);
```

and recommends to replace it with:

```cs
text = text[..(17 + 4)];
```

Would you add it here too please?

Reply:
I opened #6934 for that and for similar warnings.

Reply:
tx